scout/adapter/mongo/case_events.py

```diff
@@ -388,10 +388,13 @@ def request_rerun(self, institute, case, user, link):
         Return:
             updated_case
         """
-        if case.get("rerun_requested"):
-            raise ValueError("rerun already pending")
+        rerun_status = True
```

Comment:
Renaming this var could make it an easier read. The toggle part next is a bit dense on first read, especially with the function name still being request_rerun. Perhaps the function should also simply be renamed `update_rerun_status`, like the endpoint? With a new name and docstring the var name is more obvious.

Reply:
I agree. Let's modify the function name, since the function purpose is different now!

Reply:
Renamed also variable to is_rerun